Tests/appstoreconnect-cliTests/Operations/GetUserInfoOperationTests.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+@testable import AppStoreConnectCLI
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+import XCTest
+
+typealias Dependencies = GetUserInfoOperation.Dependencies
+typealias Options = GetUserInfoOperation.Options
+typealias OperationError = GetUserInfoOperation.GetUserInfoError
```

Comment:
Should we move these `typealias` into `final class GetUserInfoOperationTests`? I guess these types only need to be accessed by `GetUserInfoOperationTests`